.travis.yml

```diff
@@ -8,7 +8,7 @@ install:
 
 script:
   - (cd test && ./runtests.sh) # I/O tests
-  - ./gradlew -b build.gradle.txt clean test # JUnit tests
+  - ./gradlew -b build.gradle.txt clean test check # JUnit tests
```

Comment:
`check` includes `test` as well, so `test` is redundant if we are running `check`.

Also, is the comment "JUnit tests" still correct if we are running checkstyle as well?

Reply:
Actually let's change `check` to `checkstyleMain` and `checkstyleTest`. `check` itself is an aggregate task that depends on `test` so I thought it would be better to be more explicit. Better to leave no room for confusion.

Reply:
> check itself is an aggregate task that depends on test so I thought it would be better to be more explicit. Better to leave no room for confusion.

Actually, it will confuse students more since they would assume that `check` will not run `test`.

Also, as a general rule we always write code that is succinct, because: (1) It demonstrates that you understand what is going on, and (2) so we write code which do the same thing the same way (code style standardization).